index.bs

```diff
@@ -710,6 +711,11 @@ A [=sensor type=] has a [=permission revocation algorithm=].
             1.  Invoke [=revoke sensor permission=] with |sensor| as argument.
 </div>
 
+A [=sensor type=] has an associated [=policy-controlled feature=] which is called <dfn>sensor feature</dfn>.
```

Comment:
`and an associated [=feature name=] which is called <dfn>sensor feature name</dfn>.`